### PR TITLE
add a few explanation for slow logs

### PR DESCRIPTION
**Pull request by ekexium**
Signed-off-by: ekexium <ekexium@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add a few explanations for some time metrics.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@ekexium 是否为 v6.0 相关文档呢？请@ tech review，谢谢~

**Comment by ekexium** [comment]
> @ekexium 是否为 v6.0 相关文档呢？请@ tech review，谢谢~

不相关，只是补足文档不清晰的一些地方。这些参数应该是 5.0 开始就有的。

File: sql-statements/sql-statement-explain-analyze.md [hunk 1]
```diff
@@ -84,16 +84,16 @@ EXPLAIN ANALYZE SELECT * FROM t1 WHERE id = 1;
 {{< copyable "sql" >}}
 
 ```sql
-EXPLAIN ANALYZE SELECT * FROM t1;
+EXPLAIN ANALYZE SELECT * FROM t;
```
Comment (sticnarf) [inline sql-statements/sql-statement-explain-analyze.md:87]: The table name created above is `t1`. It should remain unchanged.

**Review by sticnarf — COMMENTED** [review]
The rest lgtm

**Review by sticnarf — APPROVED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**